Modules/Setup.stdlib.in

```diff
@@ -131,6 +130,14 @@
 @MODULE__TESTBUFFER_TRUE@_testbuffer _testbuffer.c
 @MODULE__TESTINTERNALCAPI_TRUE@_testinternalcapi _testinternalcapi.c
 
+
+############################################################################
+# Template modules
+
+@MODULE__XXSUBINTERPRETERS_TRUE@_xxsubinterpreters _xxsubinterpretersmodule.c
+@MODULE_XXLIMITED_TRUE@xxlimited xxlimited.c
+@MODULE_XXLIMITED_35_TRUE@xxlimited_35 xxlimited_35.c
```

Comment:
I think these modules must be build as shared modules, too. They use the limited API.